src/openrct2/paint/track/coaster/SingleRailRollerCoaster.cpp

```diff
@@ -18391,6 +18391,51 @@ namespace OpenRCT2::SingleRailRC
         TrackLeftEighthBankToDiagUp25(session, ride, trackSequence, (direction + 3) & 3, height, trackElement);
     }
 
+    static void Track25DegDownBrakes(
```

Comment:
Can you change this function to use the hybrid coaster's Track25DegDownBrakes as the basis? It helps unify the code which will help making track paint into objects.